modules/virt-creating-configmap.adoc

```diff
@@ -150,13 +250,13 @@ parameters:
 EOF
 ----
 
-. Verify that the Windows template was created by running the following command:
+.. Verify that the Windows template was created:
 +
 ----
 $ oc get template -A | grep win
 ----
 
-. Create a custom ConfigMap based on the following example:
+.. Create a ConfigMap to map the RHV VM Windows operating system to the Windows template:
```

Comment:
Wait -- are they supposed to create the ConfigMap in step 4 *and* another ConfigMap here?

Reply:
No. It's a series of conditions. IF their OS has a corresponding template, they create a config map earlier. For Windows OS, they MUST create a configmap here because a bug prevents the existing Windows templates from being used. (Import fails because the template creates a VM first, without a disk, and that's not supported. So they have to update the Windows template for 2.4.1.)